android/README.md

```diff
@@ -74,115 +43,130 @@ allprojects {
 ```
-**Developer note**: `serviceTrigger.run(context)` will run inside a pooled thread created using by Executors API. Please be aware of it.
 
-### Scheduler
 
-To be able to run the jobs periodically even after the device has rebooted you can use GCM scheduler in that case. To be able to do that you need to update your `build.gradle` with the following dependencies:
+## Scheduled Trigger
+You can let the SDK manage the scheduling of the DDC calls. It's only required to initialize it.
+Scheduling is built on GCM network manager [https://developers.google.com/cloud-messaging/network-manager](https://developers.google.com/cloud-messaging/network-manager)
+Scheduled jobs will continue to run even when the host app is not running or is put in background. Frequency is defined in the licence.
+
+### Project Setup
+Two extra dependencies are required in order to use the scheduler: *io.ebuilder.mobile.services.scheduler.gcm:ddc-gcm-scheduler* and *com.google.android.gms:play-services-gcm*
+
+Adjust your *build.gradle* as following:
 
 ```groovy
 dependencies {
-    compile "com.google.code.gson:gson:2.7"
-    compile "org.apache.commons:commons-lang3:3.5"
-    
-    compile "io.ebuilder.mobile.services:ddc-sdk:x.y.z@aar"
-    compile "io.ebuilder.mobile.services.scheduler.gcm:ddc-gcm-scheduler:x.y.z@aar"
-    compile "com.google.android.gms:play-services-gcm:11.6.0"
+    implementation 'com.android.support:support-v4:27.1.1'
+    implementation 'com.google.code.gson:gson:2.8.4'
+    implementation 'org.apache.commons:commons-lang3:3.5'
+    implementation 'com.google.android.gms:play-services-gcm:15.0.1'
+    implementation 'io.ebuilder.mobile.services:ddc-sdk:1.2.0.156@aar'
+    implementation 'io.ebuilder.mobile.services.scheduler.gcm:ddc-gcm-scheduler:1.1.0.19@aar'
 }
 ```
 
-To setup the scheduler please use the following code:
+### Initialization
+Initialization is similar to the On Demand Trigger having same mandatory and optional settings:
 
-```java
-final SchedulerSettingsBuilder builder = (SchedulerSettingsBuilder)ScheduledDDCFactory.setup(this, SYSTEM_ID, telephonyManager.getDeviceId(), DeviceIdType.IMEI)
-    .loggingEnabled()
-    .collectGoogleAccounts();
-builder.scheduler(this, ScheduledLicense.class).reschedule(this);
+### Usage
+Obtain a reference of the scheduler:
 
+```java
+Scheduler<ScheduledLicense> scheduler = ScheduledDDCFactory.setup(this, SYSTEM_ID, "deviceId", DeviceIdType.IMEI)
+                .loggingEnabled().scheduler(this, ScheduledLicense.class);
 ```
 
-If you no longer want to use the scheduler you can unschedule it:
+Then you can schedule the jobs (*scheduler.reschedule(Context context)*):
 
-```
-final SchedulerSettingsBuilder builder = (SchedulerSettingsBuilder)ScheduledDDCFactory.setup(this, SYSTEM_ID, telephonyManager.getDeviceId(), DeviceIdType.IMEI)
-    .loggingEnabled()
-    .collectGoogleAccounts();
-builder.scheduler(this, ScheduledLicense.class).cancel(this);
+```java
+scheduler.reschedule(this);
```

Comment:
Maybe write a few lines about it not being dangerous to call re-schedule to avoid the same misconceptions we had integrating with Kurashiru